### PR TITLE
Update and reorganize final-version.md

### PR DESCRIPTION
**Pull request by davidweichiang**
I updated and reorganized final-version.md. However, I think it's odd to have formatting.md which covers both review and final versions and final-version.md which covers just final versions.

Maybe some sections of formatting.md (anonymity, multiple submission, rulers) should be separated out into review-version.md?

### CODE REVIEW

**Comment by davidweichiang** [comment]
There are still some overlaps between start.md and final-version.md. I will work on it more tomorrow.

**Comment by mjpost** [comment]
I guess the question is whether you want to rebase on the split I pushed to master between formatting.md and formatting-start.md (which name I don't love—but start.md contains unmerged instructions for pub chairs).

I'll leave the edit token with you unless I hear otherwise.

**Comment by davidweichiang** [comment]
Will do. Questions: 

- Did you make any improvements to final-version.md when you embedded it into formatting.md? 
- How do I preview the web pages locally?

**Comment by mjpost** [comment]
* I did make some improvements. [In the diff](https://github.com/acl-org/ACLPUB/commit/a370f92c742f994a0d094ec5144efb96b89d4973), but they're not clean commits. See:
   * In `formatting.md`, see "What should I know about graphics?" which consolidates a few things
   * In the new `formatting-start.md`, the questions "How should I record my name in the START system?" and "What about other metadata?"

* To view locally:
   * Install [jekyll](https://jekyllrb.com)
   * Edit `_config.yaml`, I think you'll want "baseurl: /"
   * From within docs, run `bundle exec jekyll serve`

**Comment by mjpost** [comment]
I can do a proof read after you're done to see if I remember anything else I'd added.

**Comment by davidweichiang** [comment]
How does this look?

I couldn't get Jekyll to work, so was not able to preview it.

File: docs/final-version.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- Ensure that your paper conforms to the [Formatting Guidelines](formatting.html).
+- Include the authors' names and affiliations under the title.
+- De-anonymize references to your own work in the body of the paper.
+- Ensure that all [tables, graphs, and figures](formatting.html#figures-and-tables) are readable.
+- Ensure that the paper does not exceed the [page limit](formatting.html#paper-length). Most *ACL conferences allow one extra page to address reviewer comments.
+- If appropriate, add an [Acknowledgments section](formatting.html#acknowledgements).
+- Check the [paper size and embedded fonts](formatting.html#file-format).
+- If your paper's [title or other metadata](#metadata) has changed since submission, please update them so that they will appear correctly in the table of contents, author index, conference schedule, etc.
+
```
Comment (mjpost) [inline docs/final-version.md:30]: This is nice.

File: docs/review-version.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+> Gusfield (1997) previously showed...
+
+Review versions must not include acknowledgements.
+
+**Papers that do not conform to these requirements may be rejected without review.**
+
+Any preliminary non-archival versions of submitted papers should be listed in the submission form but not in the review version of the paper.
+Reviewers are generally aware that authors may present preliminary versions of their work in other venues, but will not be provided the list of previous presentations from the submission form.
+
```
Comment (mjpost) [inline docs/review-version.md:46]: This was a bit confusing to me. I think you are referring to the paper submission process, but whether this is possible varies by venue and venue policy.

**Review by mjpost — APPROVED** [review]
This looks good. I like the decomposition.

File: docs/review-version.md (hunk 2 above)
Comment (davidweichiang) [reply docs/review-version.md:46]: Hm, I agree it's confusing, but this policy comes from the existing formatting guidelines.

To unconfuse it, I suggest changing the first line to "should be listed in the submission form, but not in the paper itself."

Comment (mjpost) [reply docs/review-version.md:46]: That's clearer.

**Comment by mjpost** [comment]
Here's a preview, btw: http://www.cs.jhu.edu/~post/tmp/aclpub/

**Comment by davidweichiang** [comment]
Does anyone know what the text is for the START link to submit a final version of a paper? I wrote "Submit final version" but that was just a guess.

**Comment by mjpost** [comment]
image

**Comment by mjpost** [comment]
Here's the submission page for WMT, for reference: http://cs.jhu.edu/~post/tmp//Users/post/Desktop/submit.html

**Pull request merged**